lib/p2p/packets/types/OrderInvalidationPacket.ts

```diff
@@ -0,0 +1,20 @@
+import Packet, { PacketDirection } from '../Packet';
+import PacketType from '../PacketType';
+
+type OrderInvalidationPacketBody = {
+  orderId: string;
+  pairId: string;
+  // quantity: number;  TODO: implement quantity invalidation
```

Comment:
Which quantity will be sent in the `OrderInvalidationPacketBody`? How much was filled or how much is remaining?

Reply:
How much was filled, because that's what you are invalidating. Unless we'll find it safer/easier to do otherwise.